packages/ui-signer/src/Extrinsic.tsx

```diff
@@ -40,7 +40,7 @@ class Transaction extends React.PureComponent<Props> {
           <div className='expanded'>
             <p>
               <Trans i18nKey='decoded.short'>
-                You are about to sign a message from <span className='code'>{accountId}</span> calling <span className='code'>{section}.{method}</span> with an index of <span className='code'>{accountNonce.toString()}</span>
+                You are about to sign a message from <span className='code'>{accountId}</span> calling <span className='code'>{section}.{method}</span> with an index of <span className='code'>{accountNonce ? accountNonce.toString() : 0}</span>
```

Comment:
Still worried about this `accountNonce ? ` - it should be set here. If not, we have an underlying bug.

Reply:
Moved this to a separate issue so it can be tracked. #450